tests/db/test_db_schema.py

```diff
@@ -36,6 +36,7 @@
 from utils import log_contains
 
 
+@pytest.mark.slowtest
```

Comment:
I didn't realize that the db directory contained other test files (like `tests/db/test_db_schema.py` and `tests/db/test_restore_tool.py`) next to the migration test files. These former aren't slow so they shouldn't get the slowtest annotation. Maybe we should move all database migration related test files to a separate subdirectory to separate them from the other database related test files.

Reply:
I'll do that :)